app/controllers/application_controller.rb

```diff
@@ -1,10 +1,28 @@
 require './config/environment'
+require 'rack-flash'
 
 class ApplicationController < Sinatra::Base
 
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "secret"
+    use Rack::Flash
   end
 
-end
\ No newline at end of file
+  get '/' do
+    @title = "Fwitter - Welcome"
```

Comment:
Using the variable for title allows me to customize the tab for the current page a user is visiting.